### PR TITLE
Refactor members and implement viewing

### PR DESCRIPTION
**Pull request by Macroz**
See #609 and #870.

- Removes add member support for old applications
- Splits add member events to add member (REMS user, handler action only) and invite member (not rems user, handler and applicant depending on state)
- Implements a view of the additional members of an application as well as invited members.

### CODE REVIEW

File: resources/translations/en.edn [hunk 1]
```diff
@@ -139,6 +143,7 @@
                                   :decision-requested "Decision requested"
                                   :draft-saved "Modified"
                                   :member-added "Member added"
+                                  :member-invited "Invited member"
```
Comment (cscwooller) [inline resources/translations/en.edn]: This is quite splitting hair, but would prefer "Member invited" for consistency.

File: resources/translations/fi.edn [hunk 2]
```diff
@@ -139,6 +143,7 @@
                                   :decision-requested "Pyydetty päätöstä"
                                   :draft-saved "Muokattu"
                                   :member-added "Lisätty jäsen"
+                                  :member-invited "Kutsuttu jäsen"
```
Comment (cscwooller) [inline resources/translations/fi.edn]: Same here, as an event "Jäsen kutsuttu"? Although here the member-added is not in same form.

File: src/clj/rems/db/applications.clj [hunk 3]
```diff
@@ -946,17 +940,6 @@
...
-  (let [application (get-application-state application-id)]
-    (when-not (= user-id (:applicantuserid application))
-      (throw-forbidden))
-    (when-not (#{"draft" "returned" "withdrawn"} (:state application))
-      (throw-forbidden))
-    (assert (users/get-user-attributes member) (str "User '" member "' must exist"))
-    (db/add-application-event! {:application application-id :user user-id :round 0
-                                :comment nil
-                                :event "add-member" :eventdata (cheshire/generate-string {"uid" member})})))
-
```
Comment (cscwooller) [inline src/clj/rems/db/applications.clj]: Good that we are getting code away from applications. :)

File: resources/translations/en.edn (hunk 1 above)
Comment (Macroz) [reply resources/translations/en.edn]: Yes it's better like that. Matches the others.

File: resources/translations/fi.edn (hunk 2 above)
Comment (Macroz) [reply resources/translations/fi.edn]: In this case the other Finnish translation matches this form and I think both ways are fine so I'll keep it like this.

**Pull request merged**